### DISCUSSION TITLE
How the TMS coil orientation is determined automatically when using the "project to scalp" function?

### DISCUSSION THREAD

**Discussion by SoraAN55** [simnibs/simnibs#142] [2. General]
Hello,

I am a beginner who has just started using simnibs.

I built an individual head model using "headreco", and obtained target coordinates in subject space (in the brain) using "mni2subject_coords" to target a specific location in MNI space.

I used the "project on scalp" option to place the TMS coil, and confirmed that the coordinates are automatically scalp projected.
At this time, I was confirmed that not only the center position of the coil but also the orientation of the coil (i.e., y-axis) was set automatically.

I wonder how the orientation of the coil is determined automatically.
I've understood that the center position is projected to the nearest scalp position from the target coordinates, but I haven't understood how the coil's orientation is determined.

Could you please explain regarding this a bit more detail or let me know some reference material?

Thank you in advance.

**Comment by oulap** [comment]
Hi,

I assume that you're talking about the GUI here. Once you define the center of the coil, the direction is automatically set by (somewhat) arbitrarily adding 10 to the y coordinate of the center coordinate. This is just an initialization to make sure a direction is defined. To define your own direction you can then double-click on the head surface, or select the direction based on the EEG cap layout. 

If you're using a script to do the simulations you have to set the coil direction or the simulation will not run.
